section2-week1/exercise7/Main.cpp

```diff
@@ -0,0 +1,103 @@
+/*Exercise 7 - Rock, Paper, Scissors
+Write a simple rock, paper, scissors game.
+Ask the user for their guess on the result, then tell them whether they guessed right or wrong.
+*/
+
+#include <iostream>
+#include <stdio.h>      /* printf, scanf, puts, NULL */
+#include <stdlib.h>     /* srand, rand */
+#include <time.h>       /* time */
+using namespace std;
+
+int main()
+{
+	int value, v1, ans;
+	ans = 0;
+
+	srand(time(NULL));
+	v1 = rand() % (1 - 4);
+
+	cout << "Hey you, let's play Rock, Paper, Scissors!" << endl;
+
+	cout << "1 - Rock\n2 - Paper\n3 - Scissors" << endl;
+	cout << "choose you weapon: " << endl;
+
+	cin >> value;
+
+	cout << "\nI have selected the weapon: " << v1 << endl;
+
+
+	//comparing the weapons
+	switch (value) 
+	{
+		case 1:
+			cout << "You have selected Rock" << endl;
+			if (v1 == 1) 
```

Comment:
You're executing all your if's here. You could have used else if's to avoid this, but in this case, a switch would be a much better choice.